pydis_core/_bot.py

```diff
@@ -19,7 +18,7 @@
     from async_rediscache import RedisSession
     REDIS_AVAILABLE = True
 except ImportError:
-    RedisSession = None
+    RedisSession = object
```

Comment:
Had to do this as otherwise we'd have `NoneType | NoneType` if redis couoldn't be imported, which will raise an error